### DISCUSSION TITLE
请问fitz的block里的文本的坐标，跟paddleOcr坐标识别不一致怎么办？

### DISCUSSION THREAD

**Discussion by zxf1412** [PaddlePaddle/PaddleOCR#13684] [Q&A]
我先用fitz，做一个pdf的文字提取，获取文本和坐标。然后再用paddleOcr的layout获取文档的结构，然后做一个匹配。
但是fitz获取的一行文字的坐标和paddleOcr的坐标，不一致。怎么办，paddleOcr有什么参数，可以控制坐标大小吗。

**Comment by UserWangZz** [comment]
这个可以用IoU匹配吗？
